### DISCUSSION TITLE
Footer line shown since one of the latest version

### DISCUSSION THREAD

**Discussion by renrom** [Budibase/budibase#11794] [Help] [answered]
Hi, i noticed a footer line since one of the latest versions:

![image](https://github.com/Budibase/budibase/assets/26928569/e6cf34e6-d7c9-4cb6-88f7-bdf4291b3a76)

Is that shown because i use the local version of budibase? Is it possible to hide this line? or will that be a pay thing?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @renrom,

This I believe this has always been around in some from. You likely just missed it in previous versions. (It may have been a little more subtle previously.

Yes in order to hide Budibase's branding I believe its a Business plan feature.

I hope this helps.

**Reply by renrom** [reply]
I never seen it before, just after updated to the latest version i noticed and i never that far behind with versions. maybe it was indeed more subtle before ;-). to be honest, i think it is a waste of space, as the people who work with this already know its made in budibase.

**Reply by ConorWebb96** [reply]
Thanks for the feedback! I believe this is more aimed at external users whom don't know its made with Budibase.
